ansible-ipi-install/roles/installer/templates/install-config.j2

```diff
@@ -30,7 +30,9 @@ platform:
   baremetal:
     apiVIP: {{ apivip }}
     ingressVIP: {{ ingressvip }}
+    {% if (release_version[0]|int < 4) or ((release_version[0]|int == 4) and (release_version[2]|int < 5)) %}
```

Comment:
We don't have a release less than 4 so no need on the first check, I think all we need is

```
{% if ((release_version[0]|int == 4) and (release_version[2]|int < 5)) %}
```